docs/source/ui/pages/run_management.md

```diff
@@ -189,3 +196,140 @@ Use the `Settings` tab to delete or archive the single run
 
 <img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Archive card" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/run-single-page/archive_card.png">
 <img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Unarchive card" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/run-single-page/unarchive_card.png">
+
+## Experiment page
+
+The experiment page provides overall information about the single experiment, and it helps you to group related runs and explore them.
+
+Here are the tabs available on the experiment page.
+
+- [Overview](#id21)
+- [Runs](#id24)
+- [Notes](#id25)
+- [Settings](#id26)
+
+### Overview
+
+The Overview tab provides an overview of the runs that are attached with the experiment.
```

Comment:
The Overview tab provides an overview of the runs that are attached **to** the experiment.